docs/platform/concepts/orgs-units-projects.md

```diff
@@ -2,27 +2,31 @@
 title: Organizations, units, and projects
 ---
 
-The Aiven platform uses organizations, organizational units, and projects to efficiently and securely organize your services and manage access.
-
 import Overview from "@site/static/images/content/figma/organizations-overview.png";
 
-<img alt="Hierarchy showing two organizational units, each with two projects, nested within one organization." src={Overview} width="58%" className="centered"/>
+The Aiven platform uses organizations, organizational units, and projects to efficiently and securely organize your services and manage access.
 
 There are three levels in this hierarchy:
 
-1. Organization: Contains all of your projects and services. It's recommended to have one
+- **Organization**: Contains all of your projects and services. It's recommended to have one
     Aiven organization.
-1. Organizational units: Added to the organization, giving you greater flexibility to
+- **Organizational units**: Added to the organization, giving you greater flexibility to
    organize your infrastructure based on your specific use cases. For example, you can
    split production and testing workloads into different organizational units.
-1. Projects: Created in the organization or organizational units to group your services
+- **Projects**: Created in the organization or organizational units to group your services
     together.
 
+<div style={{ textAlign: 'center' }}>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/axhcppn6W2c?si=6uC3mmQ4ATNbyTrt" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
Somehow, fullscreen only works when capitalized... 🫠 😆 
```suggestion
  <iframe width="560" height="315" src="https://www.youtube.com/embed/axhcppn6W2c?si=6uC3mmQ4ATNbyTrt" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowFullScreen></iframe>
```

Reply:
Weird, I copied it from YT. 😄

Reply:
I know 😆 I'm not sure I remember why this is case sensitive but I just know that's the only gotcha  (that I forgot to mention when we talked about it) 😊